installation-usage.md

```diff
@@ -932,6 +932,29 @@ security_profiles_operator_seccomp_profile_audit_total{container="log-container"
 security_profiles_operator_seccomp_profile_audit_total{container="log-container",executable="/usr/sbin/nginx",namespace="default",node="127.0.0.1",pod="log-pod",syscall="write"} 20
 ```
 
+## Configuring webhooks
```

Comment:
The linter won't be happy with this change without changing the table of contents.

Reply:
Thanks, done